components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -354,7 +422,12 @@ const WorkspaceOptimizationDashboard = (props: Props) => {
                 defaultMessage='Workspace Optimization'
             />
             <div className='admin-console__wrapper'>
-                <OverallScore chips={<ChipsList chipsData={overallScoreChips}/>}/>
+                <OverallScore
+                    chips={<ChipsList chipsData={overallScoreChips}/>}
+
+                    // temp to see either the alert image or the circular chart
+                    chartValue={Math.floor(Math.random() * 100) + 1}
```

Comment:
random number, if below 50, shows the overal score image red icon, if over, shows the circular chart, so @michaelgamble can see both in action. Either way, I guess we will calculate this value here in the dashboard controller and pass it as a value to the OverallScore component.

Reply:
well, this is definitely temporal, we were hardcoding a number value before, so having a random number doesn't make any difference. I added a big TODO to remove it, but we need to define how are we going to calculate the value we pass to the overall score chart, including what is the threshold to show the red alert icon.